test/e2e/catalog_e2e_test.go

```diff
@@ -31,7 +31,7 @@ import (
 var _ = Describe("Catalog", func() {
 	It("loading between restarts", func() {
 
-		defer cleaner.NotifyTestComplete(GinkgoT(), true)
+		defer cleaner.NotifyTestComplete(true)
```

Comment:
Shouldn't this use the `AfterEach` Ginkgo feature? Applies in all places

Reply:
Same explanation as above

Reply:
Oops